### PR TITLE
Added pip install theseus-ai instructions.

### PR DESCRIPTION
**Pull request by luisenp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,15 @@ We support several features that improve computation times and memory consumptio
     - `conda install -c conda-forge suitesparse` (Mac).
     
 ### Installing
+#### **pypi**
+```bash
+pip install theseus-ai
+```
+We currently provide wheels with our CUDA extensions compiled using CUDA 10.2 and Python3.9.
```
Comment (mhmukadam) [inline README.md:79]: nit: Python3.9 -> Python 3.9

Comment (luisenp) [reply README.md]: Fixed.

File: build_scripts/build_wheel.sh [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+# -----------------
+# To use this script, from root theesus folder run 
+#    ./build_scripts/build_wheel.sh ROOT_DIR TAG CUDA_VERSION
+#
+# ROOT_DIR: is the directory where the tar.gz and .whl files will be stored
+# TAG: is a theseus tag (e.g., 0.1.0)
+# CUDA_VERSION: the version of CUDA to use. We have tested 10.2, 11.3, and 11.6
+# -----------------
+
+# Ensure that 3 arguments (ROOT_DIR, TAG, CUDA_VERSION) are provided
```
Comment (mhmukadam) [inline build_scripts/build_wheel.sh:12]: might be nice to give a fully instantiated example here as well

Comment (luisenp) [reply build_scripts/build_wheel.sh:22]: Added.

**Review by mhmukadam — APPROVED** [review]

**Pull request merged**